loom-cli/main/loom.hs

```diff
@@ -227,8 +229,56 @@ buildLoom' logger buildConfig mode config home sitePrefix apx sf (BuildConfig ha
   withLogIO logger "site" . firstT LoomSiteError $
     generateLoomSite sitePrefix siteRoot apx sf r
 
+
+bundlePatterns ::
+  Loom ->
+  HardcodedBundle ->
+  PathSet ->
+  [FilePattern]
+bundlePatterns (Loom cfg cfgs) bundle sets =
```

Comment:
I'm wondering if it's worth keeping the main file pretty streamlined if/when we add more commands? I don't really have a good suggestion about where to move this to though, possibly in "loom-purescript", but otherwise just a purescript module in "loom-cli".